examples/common/python/crypto/crypto.i

```diff
@@ -106,6 +107,7 @@ namespace std {
 
 %ignore ByteArrayToString;
 
+%include "utils.h"
```

Comment:
Do we need to expose python wrapper for functions in utils.h ?

Reply:
i have removed unwanted header files inclusion.